File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/menu/MenuGroupPortionEntity.java
package com.robinfood.core.entities.menu;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class MenuGroupPortionEntity {

    private final BigDecimal discount;
    private final Long id;
    private final String image;
    private final Boolean isDefault;
    private final String name;
    private final Long parentId;
    private final BigDecimal premiumPrice;
    private final BigDecimal price;
    private final Integer position;
    private final String sku;
    private final Long unit;
    private final Double weight;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/menu/MenuGroupPortionEntity.kt
package com.robinfood.core.entities.menu

import java.math.BigDecimal

data class MenuGroupPortionEntity(
    val default: Boolean?,
    val discount: BigDecimal?,
    val id: Long?,
    val image: String?,
    val name: String?,
    val parentId: Long?,
    val premiumPrice: BigDecimal?,
    val price: BigDecimal?,
    val position: Int?,
    val sku: String?,
    val unit: Long?,
    val weight: Double?
)